service_implementation/advanced_plm.md

```diff
@@ -1 +1,455 @@
-# Beyond just files: Using PLM for versioning and metadata
\ No newline at end of file
+# Beyond just files: Using PLM for versioning and metadata
+This document describes how to use PLM services from _inside_ a service, i.e., by calling the PLM API. The text includes  example code.
+There are two types of PLM services for use in CAxMan; both are described below:
+1) those that are integrated in the GSS framework;
+2) the PLM Server native web-services.
+
+## GSS
+
+The PLM Server has its own implementation of the GSS services. GSS offers SOAP and REST services:
+- WSDL (SOAP): https://caxman.clesgo.net/jotne/GSS/FileWebService?wsdl
+- REST end point: https://caxman.clesgo.net/jotne/GSS/resources/plm
+
+GSS covers a small subset of simplified methods (services) of the PLM Server.
+
+Java (Maven) client based examples can be retrieved from here: https://github.com/CAxMan/PlmGSS-samples
+
+The PLM Server is also accessible via the GSS implementation by SINTEF.
+Code samples of that are here: https://github.com/CAxMan/infrastructureClients/tree/master/gssClients

```

Comment:
Please link to `https://github.com/CAxMan/infrastructureClients` instead, since there is one new generic infrastructure library.

Reply:
fixed